Modules/_sqlite/module.c

```diff
@@ -346,6 +346,16 @@ static struct PyModuleDef _sqlite3module = {
         NULL
 };
 
+#define ADD_OBJECT(module, name, type)                             \
```

Comment:
```suggestion
#define ADD_TYPE(module, type)                             \
```

use PyModule_AddType :)